app/views/profiles/show.html.erb

```diff
@@ -20,6 +20,15 @@
             class: "t-link--black"
           )
         %>
+        <% if @user.verified %>
+          <%=
+            image_tag(
+              "rubygems_logo_black.png",
+              size: 40,
+              title: 'Verified User'
+            )
+          %>
+        <% end %>
```

Comment:
Could we just have text that shows something like:

> ✅ Verified

Instead of reusing the logo here?

Reply:
The existing logo was easier to use (and I personally like a gem icon over an actual checkmark!), but I'm open to using whatever.